allennlp/predictors/predictor.py

```diff
@@ -57,17 +63,157 @@ def predict_json(self, inputs: JsonDict) -> JsonDict:
         instance = self._json_to_instance(inputs)
         return self.predict_instance(instance)
 
+    def interpret_from_json(self, inputs: JsonDict) -> JsonDict:
+        """
+        Uses the gradients from :func:`get_gradients` to provide 
+        normalized interpretations for specific models. 
+
+        Raises
```

Comment:
We haven't included these warnings in our docs, particularly for something like this.  I'd just remove it (same for other methods).  You could put a note in the docstring that says subclasses must implement this method if they want this functionality (without the separate `Raises` block).